VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.spec.ts

```diff
@@ -83,6 +83,7 @@ describe('JudgeParticipantStatusListComponent', () => {
 
         expect(component.staffMembers).toBeDefined();
         expect(component.staffMembers.length).toBe(1);
+        expect(component.staffMembers[0].display_name).toBe('Staff Member');
```

Comment:
can you change this to a const please, we should stop testing text in template files

Reply:
this is the hardcoded constant in the test data

Reply:
not testing text it is test data.